index.html

```diff
@@ -33,6 +33,8 @@
                     <li class="nav-item"><a class="nav-link" href="supermarkets.html">Supermarkets nearby</a></li>
                 </ul>
             </div>
+
+            <button type="button" class="btn btn-warning" data-toggle="modal" data-target="#cart"> Shopping list </button>
```

Comment:
you don't need to place spaces around "Shopping list" words. Because text which is written between tags will be displayed exactly as you write it in your code.